src/Libraries/Notifications/Notifications/Notification.cs

```diff
@@ -32,39 +32,40 @@
 using EventArgs = System.EventArgs;
 
 namespace Notifications {
```

Comment:
I thought the enum braces were wrong when I first looked but it's the namespaces and it's everywhere in the codebase, oh the horror! :)

I am kidding - but do we need an addition to the R# style rules?

Reply:
Good catch :wink: 

R# settings seem fine. I just didn't run the auto formatting.